### ISSUE TITLE
Bipolar temporal impulse response

### ISSUE THREAD

**Issue by wandell** [isetbio/isetbio#189]
The current IR for the bipolars is derived for a fixed cone impulse response and with a fixed retinal ganglion cell impulse response.  It was chosen (by a script) to convert the fixed cone response to the RGC impulse response.

At present, however, the cone photocurrent impulse response depends on the stimulus condition (e.g., mean level) and also cone type.  

We need to think about what we want to do here.  The RGC impulse also must vary with stimulus conditions and cell type.  In the very short term, we will use the bipolar IR that we derived for the one set of conditions.  But before too long, we will implement a method for deriving the bipolar impulse response or we will allow users to set the response or ... something.

**Comment by npcottaris** [comment]
I was not aware that the pCurrent IR depends on cone type. I thought it only depends on background isomerization rate. But if it should depend on cone type , it appears that is currently computed for S-cones only, as line 70 of `linearFilters.m` sets the mosaic to a one-S cone mosaic.

~~~
cm = coneMosaic('os',osCM,'pattern', 2);   % single cone
~~~

Is there something else going on that I am missing ?

**Comment by wandell** [comment]
You are right, my writing was imprecise.  It depends on cone type only in the sense that the three cones are experiencing different background absorption rates.  That wasn't true a month ago.  Now we return different IRs for each cone based on its own background rate.

**Comment by npcottaris** [comment]
Yes, I thought so, but I wanted to make sure I was not missing something fundamental. Thanks !
